lib/Dialect/FIRRTL/FIRRTLTypes.cpp

```diff
@@ -1122,15 +1122,15 @@ llvm::Optional<int64_t> firrtl::getBitWidth(FIRRTLBaseType type,
           for (auto &elt : bundle) {
             if (elt.isFlip && !ignoreFlip)
               return llvm::Optional<int64_t>(None);
-            auto w = getBitWidth(elt.type);
+            auto w = getBitWidth(elt.type, ignoreFlip);
```

Comment:
Does `auto w = getWidth(elt.type);` work?  If it works, that would be more efficient.

Reply:
Good observation, it should work. Trying it out.